Userland/Libraries/LibGfx/ImageFormats/JPEGXLLoader.cpp

```diff
@@ -810,10 +810,276 @@ static ErrorOr<LfChannelDequantization> read_lf_channel_dequantization(LittleEnd
 ///
 
 /// C - Entropy decoding
+class ANSHistogram {
+public:
+    static ErrorOr<ANSHistogram> read_histogram(LittleEndianInputBitStream& stream, u16 log_alphabet_size)
```

Comment:
Thanks for seeing that!
The spec never specifies the type that needs to be used, so that's just me taking a wrong guess or correcting half the plumbing when I realize what the type should be.

The value passed to this function comes from `EntropyDecoder::m_log_alphabet_size` which is already a `u8` and either default initialized to `15` or is assigned to `5 + read_bits(2)`, so that will always fits in a `u8`.